cfgov/wellbeing/urls.py

```diff
@@ -1,8 +1,14 @@
-from django.views.generic import TemplateView
 
+from core.views import TranslatedTemplateView
 from wellbeing.views import ResultsView
 
 
+try:
+    from django.urls import re_path
+except ImportError:
+    from django.conf.urls import url as re_path
+
+
```

Comment:
This already got added in a separate Django 2 support PR.

```suggestion
```

Reply:
@willbarton asked me to re-add it in a separate commit suggestion above, because he thought it was removed by mistake in one of the rebases? I just changed it yesterday.

Reply:
Yeah, I think he was partially mistaken in that suggestion he made. The `from django.conf.urls import url` on L1 needed to be removed, but the block he suggested adding was already there further down.

Regardless of what happened, it definitely doesn't need to be in there twice now.

Reply:
Yeah, I think I missed that it was already in that file (that wasn't obvious in the diff). Sorry about that!